### PR TITLE
Removed instance2instance class and files

### PR DESCRIPTION
**Pull request by devos50**
Since we will remove the wx GUI very gradually, I will open many PRs to remove parts of the GUI and refactor tests.

First, I will remove the `Instance2Instance` class. We have a fancy API now to do this stuff.

### CODE REVIEW

File: Tribler/Main/tribler_main.py [hunk 1]
```diff
@@ -950,14 +911,6 @@ def run(params=[""], autoload_discovery=True, use_torrent_search=True, use_chann
         installdir = determine_install_dir()
 
         if not ALLOW_MULTIPLE and process_checker.already_running:
```
Comment (devos50) [inline Tribler/Main/tribler_main.py]: (Laurens): This only does a log now, is it still relevant to log this?

Comment (devos50) [inline Tribler/Main/tribler_main.py]: Yes since Tribler is not starting up and you want to let users know why this is.

**Comment by devos50** [comment]
@whirm can you take a look?

File: Tribler/Main/tribler_main.py (hunk 1 above)
Comment (lfdversluis) [inline Tribler/Main/tribler_main.py]: Users do not look at the log, do they? Shouldn't that broadcast something then which is sent over the evensocket?

**Comment by whirm** [comment]
retest this please

**Comment by devos50** [comment]
```
Test raised 1 unhandled Twisted exceptions, last one was: Unhandled Error
Traceback (most recent call last):
  File "C:\Python27\lib\threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "C:\workspace\jenkins\workspace\GH_Tribler_PR_tests_win64\tribler\Tribler\Core\Utilities\twisted_thread.py", line 55, in _reactor_runner
    reactor.run(installSignalHandlers=False)
  File "C:\Python27\lib\site-packages\twisted\internet\base.py", line 1194, in run
    self.mainLoop()
  File "C:\Python27\lib\site-packages\twisted\internet\base.py", line 1203, in mainLoop
    self.runUntilCurrent()
--- <exception caught here> ---
  File "C:\Python27\lib\site-packages\twisted\internet\base.py", line 825, in runUntilCurrent
    call.func(*call.args, **call.kw)
  File "C:\workspace\jenkins\workspace\GH_Tribler_PR_tests_win64\tribler\Tribler\Core\APIImplementation\threadpoolmanager.py", line 50, in caller
    fun(*args, **kwargs)
  File "C:\workspace\jenkins\workspace\GH_Tribler_PR_tests_win64\tribler\Tribler\Core\Libtorrent\LibtorrentMgr.py", line 482, in do_dht_check
    lt_session = self.get_session()
  File "C:\workspace\jenkins\workspace\GH_Tribler_PR_tests_win64\tribler\Tribler\Core\Libtorrent\LibtorrentMgr.py", line 186, in get_session
    if hops not in self.ltsessions:
exceptions.TypeError: argument of type 'NoneType' is not iterable
```

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
```
Test raised 1 unhandled Twisted exceptions, last one was: Unhandled Error
Traceback (most recent call last):
  File "C:\Python27\lib\threading.py", line 810, in __bootstrap_inner
    self.run()
  File "C:\Python27\lib\threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "C:\Python27\lib\site-packages\twisted\_threads\_threadworker.py", line 46, in work
    task()
  File "C:\Python27\lib\site-packages\twisted\_threads\_team.py", line 190, in doWork
    task()
--- <exception caught here> ---
  File "C:\Python27\lib\site-packages\twisted\python\threadpool.py", line 246, in inContext
    result = inContext.theWork()
  File "C:\Python27\lib\site-packages\twisted\python\threadpool.py", line 262, in <lambda>
    inContext.theWork = lambda: context.call(ctx, func, *args, **kw)
  File "C:\Python27\lib\site-packages\twisted\python\context.py", line 118, in callWithContext
    return self.currentContext().callWithContext(ctx, func, *args, **kw)
  File "C:\Python27\lib\site-packages\twisted\python\context.py", line 81, in callWithContext
    return func(*args,**kw)
  File "C:\workspace\jenkins\workspace\GH_Tribler_PR_tests_win32\tribler\Tribler\Core\Modules\channel\channel.py", line 85, in _on_channel_created
    if channel_data[u'channel'].cid != self._channel_community.cid:
exceptions.AttributeError: 'NoneType' object has no attribute 'cid'
```

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
@whirm tests passed

**Pull request merged**